### ISSUE TITLE
How to deploy gun server by Docker?

### ISSUE THREAD

**Issue by liuqixing** [amark/gun#1339]
I wrote the gun server:
`	const port = ParamUtils.getPort();
	const host = ParamUtils.getHost();
	const peers = ParamUtils.getPeers();

	const app = express();
	app.use( Gun.serve );
	app.use( express.static( __dirname ) );

	app.get( '/', ( req, res ) =>
	{
		res.send( 'Hello, this is express HTTP server' );
	} );
	app.get( '/gun', ( req, res ) =>
	{
		console.log( 'serving gun' );
		return Gun.serve;
	} );

	const server = app.listen( port, host, () =>
	{
		console.log( `Sync peer is running at http://${ host }:${ port }` );
	} );
	const gun = Gun( {
		web : server,
		peers : peers,
		localStorage : false,
	} );

	//	...
	gun.get( 'sync' ).on( ( data, key ) =>
	{
		console.log( `Server : on sync: `, data, key );
	} );`

And deploy it by Docker.
And then i wrote a client to connect to it, but it did not work. The server does not receive anything.
And client get an error code 1006
here is client code:
`		const peers = ParamUtils.getPeers( peers );
		const gun = Gun( {
			peers : peers
		} );
		gun.get( 'sync' ).put( data, ( ack ) =>
		{
		});
`

**Comment by liuqixing** [comment]
is there anybody who can help me?

**Comment by gustavcorpas** [comment]
Am I correct in that this is not an issue, but more of a question?

In case your issue is a question, those are usually well received over at the gitter chat: [gitter chat](https://app.gitter.im/#/room/#amark_gun:gitter.im)

**Comment by bmatusiak** [comment]
yes please use gitter chat,,   gun does have a [Dockerfile](https://github.com/amark/gun/blob/master/Dockerfile)

my thought is to find the docker docs about how to run a project with a Dockerfile

**Comment by bmatusiak** [comment]
```
git clone https://github.com/amark/gun.git
cd gun
docker build -t gun .
docker run -p 8765:8765 -t gun
```
then open
http://localhost:8765
